http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -602,6 +602,9 @@ private static QueuedThreadPool createExecutor(String name, int minThreads, int
             pool.start();
             pool.setStopTimeout(2000);
             pool.setDetailedDump(true);
+            if (useVirtualThreads) {
```

Comment:
Is this useful for clients? HTTP client threads are only used for managing IO, not running user code, so there should be a relatively small number.

The more important question is if we can use virtual threads in the application in combination with the synchronous HTTP client API. My guess is that this just works today, since the `execute()` API just waits for the HTTP client response to be available.